Adyen/ResponseModels/CheckoutPaymentsResponse.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Data model to contain relevant parts of `/payments` and `/payment/details` call responses.
+public struct CheckoutPaymentsResponse: Decodable, Sendable {
+    
+    // TODO: should it be string or enum?
+    public let resultCode: String
```

Comment:
I would work on an `enum` basis. Find it easier and less error prone

Reply:
As a dev I prefer enums - but keeping in mind public breaking changes - maybe having an extra case `.other(String)`